src/routes/auth.py

```diff
@@ -0,0 +1,60 @@
+import requests
+import os
+from flask import Blueprint, redirect, request
+from flask_jwt_extended import current_user, create_access_token
+
+from ..config import *
+from ..models import DiscordOAuth, User
+
+
+discord_bp = Blueprint("auth_bp",__name__, url_prefix='/discord/auth')
+
+@discord_bp.route("/login")
+def login():
+    """
+    Redirect to discord auth
+    """
+    return redirect(os.environ.get("DISCORD_OAUTH_URL"))
```

Comment:
Why is this an env var? This should also *not* be static because a `state` should be added.